assets/js/common/forms.js

```diff
@@ -220,6 +220,19 @@ function initHotjarTracking(formId) {
     trackDetailsClicks(scopedFormClass);
 }
 
+// Update the Login link to Logout if user signs in
+function updateSecondaryNav() {
+    $.ajax({
```

Comment:
Maybe we could make this AJAX call a shared function so we can use the same code in `session.js` – perhaps export this function from there, eg:

```
function getSession() {
    return $.ajax({
        type: 'get',
        url: '/user/session',
        dataType: 'json'
    });
}

getSession().then(response => {
        if (response.isAuthenticated) {
            $('.js-toggle-login').toggleClass('is-conditional-link');
        }
}
```